frontend/src/individual_certificate.tsx

```diff
@@ -244,10 +251,55 @@ export function IndividualCertificate({ id }: { id: string }) {
           );
         } else if (res.status === 404) {
           throw new Error("Kaninen kunde inte hittas.");
-        } else res.blob();
+        } else if (res.status === 200) {
+          return res.blob();
+        } else {
+          throw new Error("Något gick fel.");
+        }
+      })
+      .then((blob) => {
+        if (blob) {
+          setCertificateUrl(window.URL.createObjectURL(blob));
+          setShowSummary(false);
+          setShowComplete(true);
+        } else {
+          throw new Error("Något gick fel.");
+        }
+      })
+      .catch((error) => {
+        {
+          userMessage(error.message, "error");
+        }
+      });
+  };
+
+  // Returns the updated certificate.
+  const updateCertificate = (id: string, content: any) => {
+    fetch(`/api/certificates/update/${id}`, {
+      body: JSON.stringify(content),
+      method: "PATCH",
+      credentials: "same-origin",
+      headers: {
+        Accept: "application/pdf",
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => {
+        if (res.status === 400) {
```

Comment:
Does 400 have a specific meaning here that's essentially different from the other error, otherwise move up the other cases and let the default handle this?

Reply:
yes true, there is no specific case for 400 (any more). Removed it as you suggested.